core/domain/acl_decorators.py

```diff
@@ -72,40 +71,286 @@ def check_activity_accessible(
             activity_rights.viewable_if_private)
 
 
-def check_exploration_editable(user_id, user_actions, exploration_id):
-    """Returns a boolean to signify whether given exploration is editable
-    by the user or not.
+def check_can_edit_activity(
+        user_id, user_actions, activity_type, activity_id):
+    """Checks whether the user can edit given activity.
 
     Args:
         user_id: str. Id of the given user.
```

Comment:
Apparently this is str or None. Please update docstring accordingly.

Could you please ensure that the docstrings for user_ids are correct everywhere else? The None case is important and should not come as a surprise if someone were to modify this code in the future.